documentation/content/en/books/handbook/basics/_index.adoc

```diff
@@ -717,8 +717,8 @@ When reading the "Directory Listing" column, a `-` is used to represent a permis
 |`rwx`
 |===
 
-Use the `-l` argument to man:ls[1] to view a long directory listing that includes a column of information about a file's permissions for the owner, group, and everyone else.
-For example, an `ls -l` in an arbitrary directory may show:
+Pass the `-l` argument to man:ls[1] to view a long directory listing that includes a column of information about a file's permissions for the owner, group, and everyone else.
```

Comment:
I'd keep "use" here as more natural for an end-user audience (as opposed to developer audience).